addons/account/tests/test_ir_actions_report.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+import base64
+import io
+
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+from odoo.exceptions import UserError
+from odoo.tools import pdf
+from odoo.tests import tagged
+from odoo.tools import file_open
+
+from PyPDF2 import PdfFileReader, PdfFileWriter
+
+@tagged('post_install', '-at_install')
+class TestIrActionsReport(AccountTestInvoicingCommon):
+
+    def setUp(self):
+        super().setUp()
+        self.file = file_open('base/tests/minimal.pdf', 'rb').read()
+        self.minimal_reader_buffer = io.BytesIO(self.file)
+        self.minimal_pdf_reader = pdf.OdooPdfFileReader(self.minimal_reader_buffer)
+
+    def test_download_one_corrupted_pdf(self):
+        """
+        PyPDF2 is not flawless. We can upload a PDF that can be previsualised but that cannot be merged by PyPDF2.
+        In the case of "Print Original Bills", we want to be able to download the pdf from the list view.
+        We test that, when selecting one record, it can be printed (downloaded) without error.
+        """
+        attach_name = 'original_vendor_bill.pdf'
+
+        in_invoice_1 = self.env['account.move'].create({
+            'move_type': 'in_invoice',
+            'date': '2017-01-01',
+            'invoice_date': '2017-01-01'
+        })
+
+        self.env['ir.attachment'].create({
+            'datas': base64.b64encode(self.file),
+            'name': attach_name,
+            'mimetype': 'application/pdf',
+            'res_model': 'account.move',
+            'res_id': in_invoice_1.id,
+        })
+        test_record_report = self.env.ref('account.action_account_original_vendor_bill').with_context(force_report_rendering=True)._render_qweb_pdf(in_invoice_1.id, data={'report_type': 'pdf'})
```

Comment:
```suggestion
        test_record_report = self.env.ref('account.action_account_original_vendor_bill').with_context(force_report_rendering=True)._render_qweb_pdf(in_invoice_1.id)
```
Just to be consistent everywhere :wink: